cni/network/network.go

```diff
@@ -205,8 +205,21 @@ func (plugin *NetPlugin) Stop() {
 	logger.Info("Plugin stopped")
 }
 
-// FindMasterInterface returns the name of the master interface.
-func (plugin *NetPlugin) findMasterInterface(nwCfg *cni.NetworkConfig, subnetPrefix *net.IPNet) string {
+// findInterfaceByMAC returns the name of the master interface
+func (plugin *NetPlugin) findInterfaceByMAC(macAddress string) string {
+	interfaces, _ := net.Interfaces()
```

Comment:
Why are we not acknowledging the error here, if any ?

Reply:
added a log-- it seems like if we return an empty string, we return an error in create ep info